app/views/employment/CheckEmploymentDetailsView.scala.html

```diff
@@ -31,54 +28,91 @@
+        employmentDetails.flatMap(_.pay.flatMap(_.taxablePayToDate.map(amount => bigDecimalCurrency(amount.toString)))), dummyCall,
+        messages(getAgentDynamicContent(s"$msgKeyGeneric.hiddenText.payReceived", user.isAgent)))
+
+        val totalTaxToDate = EmploymentDataForView(messages(msgKeyGeneric + ".totalTaxToDate"),
+        employmentDetails.flatMap(_.pay.flatMap(_.totalTaxToDate.map(amount => bigDecimalCurrency(amount.toString)))), dummyCall,
+        messages(getAgentDynamicContent(s"$msgKeyGeneric.hiddenText.totalTaxToDate", user.isAgent)))
+
+        val paymentsNotOnYourP60 = EmploymentDataForView(messages(getAgentDynamicContent(s"$msgKeyGeneric.paymentNotOnYourP60",user.isAgent)),
+        convertBoolToYesOrNo(employmentDetails.map(_.pay.map(_.tipsAndOtherPayments.isDefined).getOrElse(false))), dummyCall,
+        messages(getAgentDynamicContent(s"$msgKeyGeneric.hiddenText.paymentNotOnYourP60", user.isAgent)))
+
+        val tipsAndOtherPaymentsAmount = EmploymentDataForView(messages(getAgentDynamicContent(s"$msgKeyGeneric.tipsAndOtherPayments", user.isAgent)),
+        employmentDetails.flatMap(_.pay.flatMap(_.tipsAndOtherPayments.map(amount => bigDecimalCurrency(amount.toString)))), dummyCall,
+        messages(getAgentDynamicContent(s"$msgKeyGeneric.hiddenText.tipsAndOtherPayments", user.isAgent)))
+
+        val employmentDataFields = if(isInYear) {List(employerName, payeRef, directorCeasedDate, closeCompany,
+        payReceived, totalTaxToDate, tipsAndOtherPaymentsAmount)} else {
+        List(employerName, payeRef, payReceived, totalTaxToDate, paymentsNotOnYourP60, tipsAndOtherPaymentsAmount)
+        }
 
         govukSummaryList(
             SummaryList(
-                employmentDataFields
-                    .collect { case (name, Some(v)) =>
-                        summaryListRow(
-                            key = HtmlContent(messages(s"employment.employmentDetails.$name")),
-                            value = HtmlContent(v),
-                            keyClasses = "govuk-!-width-two-thirds",
-                            actions = Seq()
-                        )
-                    }
+                employmentDataFields.collect{ case EmploymentDataForView(fieldHeading, Some(fieldValue), changeLink, hiddenText) =>
+                    summaryListRow(
+                        key = HtmlContent(fieldHeading),
+                        value = HtmlContent(fieldValue),
+                        keyClasses = "govuk-!-width-two-thirds",
+                        actions = if(!isInYear) Seq((changeLink, messages("common.change"), Some(hiddenText))) else Seq.empty
+                    )
+                }
             )
         )
-
+    }
+    @if(!isInYear){
+        @formWithCSRF(action = controllers.employment.routes.CheckEmploymentDetailsController.submit(taxYear, employmentId)) {
+            @button("common.saveAndContinue", classes = Some("govuk-!-margin-top-6"))
```

Comment:
again is this margin needed to match the prototype?

Reply:
Yep. I copied this from InterestCYA